File: 151001/Kononchuk/REST/src/REST.Publisher/Models/Entities/Issue.cs
namespace REST.Publisher.Models.Entities;

public class Issue
{
    public long Id { get; set; }
    public long? EditorId { get; set; }
    public string Title { get; set; }
    public string Content { get; set; }
    public DateTimeOffset Created { get; set; }
    public DateTimeOffset Modified { get; set; }
    
    public Editor? Editor { get; set; }
    public List<Tag> Tags { get; set; } = new();

    public List<IssueTag> IssueTags { get; set; } = new();
}

File: 151004/Karvatskaya/lab3/RV_Project/publisher/src/main/java/by/bsuir/publisher/domain/Issue.java
package by.bsuir.publisher.domain;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.OneToMany;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

import java.time.LocalDateTime;
import java.util.List;

@Entity
@Table(name = "tbl_issue")
@SuperBuilder
@NoArgsConstructor
@Getter
@Setter
public class Issue extends BaseEntity {
    @ManyToOne
    private Creator creator;

    @OneToMany(mappedBy = "issue")
    private List<IssueTag> issueTags;

    @Column(unique = true, nullable = false)
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}
